### ISSUE TITLE
move jest tests into CK + GS

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-18#16]
if you ready - i'll be able to explain this plan in details for you.

**Comment by JorgeRodr** [comment]
@atherdon tell me more about this, I'm deciding what I'm going to do next :)

**Comment by atherdon** [comment]
Ok, so main idea for tests is to save time. When server more complicated that your current example - it's usually hard to follow - what methods working well and what methods became broken after last changes.
this is why i want to add tests to endpoints of my 2 main projects: `ChickenKyiv`(recipe api server) and `Groceristar`(grocery list api server).
Another idea was to move you into a real project and see how you'll handle this.

ChickenKyiv has a great case, that shows the importance of tests. `Release 0` of that api was shipped a year ago. in the same place I started to develop `release 1` and `release 2` and right now i'm not really sure that all is working fine at previous version. and this is will be your task - by adding test coverage you'll answer if `release 0` is still working well or we should fix it

Same trouble with GS. it was a 9 month ago when i ship a current version of a project. and right now we're trying to separate server and front and transition into React. so testing endpoints became a necessary thing too

**Comment by atherdon** [comment]
Btw, this is a file with CK Api endpoints - https://github.com/ChickenKyiv/recipe-api-only/blob/master/docs/FreeMenu.md
hope i'll update it soon so it'll be more easy to read for you.

For GS we didn't have that document at this moment

**Comment by atherdon** [comment]
if you want to jump into this task and have questions to me - don't hesitate and ask them

**Comment by JorgeRodr** [comment]
Probably this is the next task I take.

**Comment by atherdon** [comment]
Ok, tell me how i can help you. i did all of this servers myself, so i can answer to questions that you might have
